task-3/src/app.js

```diff
@@ -0,0 +1,26 @@
+function isPrime(num) {
+  if (num <= 1) return false; // negatives
+  if (num % 2 === 0 && num > 2) return false; // even numbers
+  const s = Math.sqrt(num); // store the square to loop faster
+  for (let i = 3; i <= s; i += 1) {
```

Comment:
this loop is blocking on high values, it's already fine, but you might want to find a way to get rid of it? :)